.github/workflows/nightly-build.yml

```
@@ -18,6 +18,13 @@ jobs:
         with:
           python-version: 3.8
 
+      - name: Install mermaid-cli
+        run: |
+          sudo apt update
+          sudo apt install graphviz
+          sudo apt install nodejs npm
```

Same as https://github.com/pyansys/dev-guide/pull/71/files#r841613692
```suggestion
          sudo apt-get update
          sudo apt-get install nodejs npm graphviz
```